content/sigs/docs/gsod/2020/projects/document-jenkins-on-kubernetes.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: gsodproject
```

Comment:
There is no such layout. You could probably use the GSoC one safely though there might be issues with hardcoded GSoC links. You can also use a generic `project` layout though it will require manually adding some asciidoc entries instead of providing YAML metadata

Reply:
Addressed